src/app/components/MenueLink/MenueLink.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import MenueLink from './MenueLink';
+import type { MenueLinkProps } from './MenueLink';
+import type { Story } from '@storybook/react';
+
+export default {
+  title: 'Component/MenueLink',
+  component: MenueLink,
+  argTypes: {},
+};
+
+const Template: Story<MenueLinkProps> = (args) => <MenueLink {...args} />;
+
+export const Sprossenfinder = Template.bind({});
+Sprossenfinder.args = {
+  children: 'Sprossenfinder',
+  to: '/app/components/sprossenfinder',
```

Comment:
You would use a link like this in your app, but that doesn't matter here, in storybook